src/ansys/fluent/core/solver/transcript.py

```diff
@@ -0,0 +1,83 @@
+"""Module for Transcript."""
```

Comment:
@ajain-work this looks good. Can you please confer with @prmukherj because he is simultaneously working on something similar. You will need to agree a way to integrate all the changes together.

Reply:
@seanpearsonuk Yes @prmukherj informed me about changes. I shall discuss with him. Regards

Reply:
@ajain-work just noticing that this is in the solver directories specifically